.template/addons/github/.github/workflows/test.yml.tt

```diff
@@ -144,7 +144,7 @@ jobs:
     needs: unit_tests
     runs-on: ubuntu-latest
     timeout-minutes: 5
-    if: github.event_name == 'pull_request'
+    if: github.event_name == 'pull_request' && !contains(join(github.event.pull_request.labels.*.name, ','), 'wip') && !contains(github.event.pull_request.title, '[wip]') && !github.event.pull_request.draft
```

Comment:
Does it work without `join`ing? According to the Doc, this should work, no?
```suggestion
    if: github.event_name == 'pull_request' && !contains(github.event.pull_request.labels.*.name), 'wip') && !contains(github.event.pull_request.title, '[wip]') && !github.event.pull_request.draft
```

> contains(github.event.issue.labels.*.name, 'bug') returns true if the issue related to the event has a label "bug".

Reference: https://docs.github.com/en/actions/learn-github-actions/expressions#example-using-an-object-filter

Reply:
I was trying to handle labels with wip substring like `type: wip`. You can see [here](https://github.com/mosharaf13/fleet-searcher-ruby/actions/runs/5530101137) skipping join tries to run danger while [here](https://github.com/mosharaf13/fleet-searcher-ruby/actions/runs/5530211236) it skipped danger for same label

Reply:
ah got it, `type: wip` is a good example why we need `join` here!